README.md

```diff
@@ -41,6 +41,9 @@ Example usage:
     # TYPE kaiterra_particulate_matter gauge
     kaiterra_particulate_matter{microns="10"} 16
     kaiterra_particulate_matter{microns="2.5"} 15
+    # HELP kaiterra_rtvoc Total Volatile Organic Compounds (TVOC) in ppb
+    # TYPE kaiterra_rtvoc gauge
+    kaiterra_rtvoc 486
```

Comment:
WDYT about renaming this to kaiterra_total_volatile_organic_compounds ? We've spelled out the other acronyms for the metric names (e.g. PM above)

Reply:
@mhansen done, added also _ppb for unit of measure